src/app/view/components/login/login-component/login-container.component.ts

```diff
@@ -19,12 +19,12 @@ import { DotRouterService } from '../../../../api/services/dot-router-service';
         </dot-login-component>
     `
 })
-export class LoginContainer {
-    private isLoginInProgress = false;
-    private message: string;
-    private passwordChanged = false;
-    private resetEmail = '';
-    private resetEmailSent = false;
+export class LoginContainerComponent {
+    public isLoginInProgress = false;
```

Comment:
why the access of properties are public?

Reply:
All properties used in a template should be public, that is reason of the changes in this file an in others in this PR.